components/camera/camera.go

```diff
@@ -61,6 +61,7 @@ type Properties struct {
 	IntrinsicParams  *transform.PinholeCameraIntrinsics
 	DistortionParams transform.Distorter
 	MimeTypes        []string
+	FrameRate        float32
```

Comment:
Let's use float64. Generally we only use float32 and special integers when:

1. We are targeting 32bit arch machines
2. We are constrained for memory

Reply:
when changing from float32 to float64 I got an error because:

`field FrameRate *float32 `protobuf:"fixed32,5,opt,name=frame_rate,json=frameRate,proto3,oneof" json:"frame_rate,omitempty"`

I'm not using locally generated proto files anymore since the API change has been merged so I assume this is a valid error I am getting. Is it possible I cannot switch it to float64 since the proto files automatically made it a float32?

Reply:
If it's defined as a float32 in the proto then keeping it as a float32 makes sense to me!